Building docker image from exp branch fails

`make deps` fails on exp branch with

```
github.com/gorilla/mux
github.com/gorilla/pat
github.com/gorilla/websocket
github.com/mattn/go-sqlite3
github.com/drone/drone/shared/sshutil
github.com/drone/drone/server/handler
github.com/drone/drone/server/database/testdatabase
make: *** [deps] Error 123
2014/09/19 09:08:19 The command [/bin/sh -c make deps build embed install] returned a non-zero code: 2
```

--- a/shared/build/docker/client.go
+++ b/shared/build/docker/client.go
@@ -14,6 +14,7 @@ import (
 	"strings"
 
 	"github.com/dotcloud/docker/pkg/term"
+	"github.com/dotcloud/docker/pkg/stdcopy"
 	"github.com/dotcloud/docker/utils"
 )
 
@@ -208,7 +209,7 @@ func (c *Client) hijack(method, path string, setRawTerminal bool, out io.Writer)
 		if setRawTerminal {
 			_, err = io.Copy(out, br)
 		} else {
-			_, err = utils.StdCopy(out, out, br)
+			_, err = stdcopy.StdCopy(out, out, br)
 		}
 
 		errStdout <- err

